.drone.yml

```diff
@@ -1,27 +1,35 @@
 build:
   image: golang:1.5
-  environment:
-    - GO15VENDOREXPERIMENT=1
-    - GOOS=linux
-    - GOARCH=amd64
-    - CGO_ENABLED=0
   commands:
-    - go get
-    - go build
-    - go test
+    - make deps
+    - make vet
+    - make build
+    - make test
 
 publish:
+  coverage:
+    when:
+      branch: master
   docker:
-    username: drone
+    username: $$DOCKER_USER
     password: $$DOCKER_PASS
     email: $$DOCKER_EMAIL
     repo: plugins/drone-pypi
+    tag: latest
     when:
       branch: master
+  docker:
+    username: $$DOCKER_USER
+    password: $$DOCKER_PASS
+    email: $$DOCKER_EMAIL
+    repo: plugins/drone-pypi
+    tag: develop
+    when:
+      branch: develop
 
 plugin:
   name: Pypi
-  desc: Publish a package to the Python package index.
+  desc: Publish files and artifacts to the Python package index
```

Comment:
AFAIK PyPI only accepts Python modules. The previous wording was clearer, or perhaps something like "Publish a Python module to the Python package index."